packages/rpc/lib/src/utils/exception/ln_client_error.dart

```diff
@@ -0,0 +1,5 @@
+class LNClientException implements Exception {
+  String cause;
+
```

Comment:
```suggestion
  String cause;
  String code;
```

would be better also return the error code